src/api/wayfire/view.hpp

```diff
@@ -270,6 +270,7 @@ class view_interface_t : public surface_interface_t, public wf::object_base_t
      */
     virtual wlr_box get_minimize_hint();
 
+    virtual void set_minimize_hint(wlr_box hint);
```

Comment:
This is a public API, should contain a docstring. What does this function do? What coordinates does hint have? (it is output-local coordinates).